.gitignore

```diff
@@ -1,7 +1,8 @@
 .idea
 *.pyc
 htmlcov
-.coverage
+.coverage*
 docs/build/*
 NitPycker.egg-info
 dist
+.*.sw?
```

Comment:
It does, if you open one multiple time (like the reference and the new version). And it's more precise to have begin as `.*.`.